### PR TITLE
Update config.json

### PR DESCRIPTION
**Pull request by Mahabmn**
Add Arabic Workspace 
Update the version

### CODE REVIEW

File: ConversationServiceWatson/src/data/config.json [hunk 1]
```diff
@@ -2,18 +2,19 @@
 	"api_key": "$(CONV_API_KEY)",
 	"watson_config": 
 	{
-		"endpoint": "https://gateway.watsonplatform.net/conversation/api/v1/workspaces/%s/message?version=2016-09-20",
+		"endpoint": "https://gateway.watsonplatform.net/conversation/api/v1/workspaces/%s/message?version=2021-11-27",
```
Comment (daisukes) [inline ConversationServiceWatson/src/data/config.json]: I didn't test this code, but I remember that version 2021-11-27 had an issue.
Could you double-check if this works?
I think `2018-05-21` is the latest version that works with the current client implementation.

**Review by daisukes — COMMENTED** [review]
Could you check if the version of the API is okay.

**Comment by Mahabmn** [comment]
updated

**Comment by daisukes** [comment]
Thank you for updating the configuration and contributing an example!

**Comment by daisukes** [comment]
One more thing, did you submit a CLA document?
https://github.com/hulop/00Readme#how-can-i-contribute

If not, please use signoff method to clarify DCO?

Please ask me if you are not sure how to do it.

**Comment by Mahabmn** [comment]
Hello @daisukes 

please, how can I do it 

Thank you

**Comment by daisukes** [comment]
The easiest way is to send signed [CLA document](https://github.com/hulop/00Readme/blob/master/CLA/cla-individual.pdf?raw=true) to me via email.

Another way is to use git rebase function, which allow you to edit commit tree (like edit comment, add signoff, etc.). If you want to try it, please read [this document](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/changing-a-commit-message)

**Comment by Mahabmn** [comment]
Hello 

I found that I sign it and sent it to socacc@jp.ibm.com on April 2022, I share the email and document with you. 

Thank you

**Comment by daisukes** [comment]
@Mahabmn 
Thank you, but could you print the document, sign it by hand, then scan it as a pdf and send it?
Sorry for bothering you.

**Comment by Mahabmn** [comment]
@daisukes 

will sign it and share it with you soon .

**Comment by Mahabmn** [comment]
Hello @daisukes 

I share the signed file with you through email 

thank you

**Comment by daisukes** [comment]
Once your document is accepted, I will merge this PR! Thanks

**Pull request merged**